### PR TITLE
{tools}[system/system] git-lfs v3.2.0

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

File: easybuild/easyconfigs/g/git-lfs/git-lfs-3.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    'files': ['bin/git-lfs'],
+    'dirs': [],
+}
+
+sanity_check_commands = [('git-lfs', '--version')]
+
+# set up Git LFS for the user account
+modluafooter = """
+if mode() == "load" then
+    cmdout = subprocess("git lfs install")
```
Comment (boegel) [inline easybuild/easyconfigs/g/git-lfs/git-lfs-3.2.0.eb:32]: Hmm, does this need to be done on *every* load? That seems expensive...

Also, can we add the Tcl equivalent for this?

**Review by boegel — DISMISSED** [review]

File: easybuild/easyconfigs/g/git-lfs/git-lfs-3.2.0.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/g/git-lfs/git-lfs-3.2.0.eb:32]: If it's light-weight enough, I guess this is good enough... Can we also add the `modtclfooter` equivalent though?

What happens if a user switches to a different version of either `git` or `git-lfs`? Is some cleanup needed then?

Comment (lexming) [reply easybuild/easyconfigs/g/git-lfs/git-lfs-3.2.0.eb:32]: The command `git lfs install` only modifies `.gitconfig` with the following aliases
```
[filter "lfs"]
        process = git-lfs filter-process
        required = true
        clean = git-lfs clean -- %f
        smudge = git-lfs smudge -- %f
```

So it is independent of the version of `git` or `git-lfs`.

Comment (boegel) [reply easybuild/easyconfigs/g/git-lfs/git-lfs-3.2.0.eb:32]: OK, excellent. Maybe clarify that with a comment.

Are you up for looking into the `modtclfooter` equivalent?

Comment (lexming) [reply easybuild/easyconfigs/g/git-lfs/git-lfs-3.2.0.eb:32]: done in https://github.com/easybuilders/easybuild-easyconfigs/pull/15699/commits/10cd5e823f854e1ad390b5f26cfdea58961b26d4, but I cannot test if the `modtclfooter` works :wink:

Comment (boegel) [reply easybuild/easyconfigs/g/git-lfs/git-lfs-3.2.0.eb:32]: Well, sure you can, use: `eb --from-pr 15699 --module-syntax Tcl --installpath-modules /tmp/$USER/pr15699 --force`, and then test if loaded the module works? Lmod can also consume Tcl module files...

Comment (lexming) [reply easybuild/easyconfigs/g/git-lfs/git-lfs-3.2.0.eb:32]: nice! it does work :+1:

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
node3131.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/e9318f318c3a9646c49ca2f69c37d7e0 for a full test report.

**Comment by boegel** [comment]
@lexming I've removed the `modluafooter` and `modtclfooter` part because:
1) Letting the load of this module triggering a silent update of `~/.gitconfig` is not nice;
2) Using `exec git lfs config` in `modtclfooter` isn't working, and I can't seem to work out why (or how to get it to work);

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3131.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/7b2ffe7b68613158bd301ceac5a1b434 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
node3703.slaking.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz (haswell), Python 2.7.5
See https://gist.github.com/f5df1d91211b953e7406d214a11cb84d for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/3a8440cc2497862dd4983a96edbb10c3 for a full test report.

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Going in, thanks @lexming!

**Comment by lexming** [comment]
> @lexming I've removed the `modluafooter` and `modtclfooter` part because:
> 1) Letting the load of this module triggering a silent update of `~/.gitconfig` is not nice;
> 2) Using `exec git lfs config` in `modtclfooter` isn't working, and I can't seem to work out why (or how to get it to work);

> @boegel weird, I did test the tcl module with a blank `.gitconfig` and it injected the aliases just fine.

**Comment by boegel** [comment]
@lexming See my failed test report at  https://github.com/easybuilders/easybuild-easyconfigs/pull/15699#issuecomment-1208178230

**Pull request merged**